3rdparty/hal_rvv/hal_rvv_1p0/merge.hpp

```diff
@@ -0,0 +1,363 @@
+#ifndef OPENCV_HAL_RVV_MERGE_HPP_INCLUDED
```

Comment:
Need to add OpenCV license header

Reply:
Do you mean the shorter version below, or another longer version?
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```